src/components/radios/index.md.njk

```diff
@@ -118,7 +118,10 @@ In services like these, the risk that they will not be noticed is lower because
 
 ### Error messages
 
-Display an error message if none of the radios are selected.
+Display an error message if the user has not:
```

Comment:
Do we also need to provide guidance for scenarios where users fill in a field incorrectly? For more context, see the [sample image in the radios discussion](https://github.com/alphagov/govuk-design-system-backlog/issues/59#issuecomment-485778739).